packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -149,3 +158,5 @@ const Map<WebRendererMode, Map<NullSafetyMode, HostArtifact>> kDartSdkJsMapArtif
     NullSafetyMode.unsound: HostArtifact.webPrecompiledSdkSourcemaps,
   },
 };
+
+const String wasmPreviewUri = 'https://flutter.dev/wasm';
```

Comment:
Per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k

```suggestion
const String kWasmPreviewUri = 'https://flutter.dev/wasm';
```